SPECS/tidy/tidy.spec

```
@@ -2,29 +2,23 @@
 %global libname libtidy
 %global upname tidy-html5
 %global tidy_compat_headers 1
-
-Name:    tidy
-Summary: Utility to clean up and pretty print HTML/XHTML/XML
-Version: 5.7.28
-Release: 4%{?dist}
-
-License: W3C
+Summary:        Utility to clean up and pretty print HTML/XHTML/XML
+Name:           tidy
+Version:        5.7.28
+Release:        5%{?dist}
+License:        W3C
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-URL:     http://www.html-tidy.org/
-Source0: https://github.com/htacg/%{upname}/archive/%{version}.tar.gz#/%{upname}-%{version}.tar.gz
-
+URL:            https://www.html-tidy.org/
+Source0:        https://gitlab.com/sulinos/resources/tidy-html5/-/archive/5.7.28/tidy-html5-5.7.28.tar.gz
```

Please pass version as a macro.

```suggestion
Source0:        https://gitlab.com/sulinos/resources/tidy-html5/-/archive/%{version}/tidy-html5-%{version}.tar.gz
```